MetaGenSense
============

A web application framework for analysis and visualization of High throughput Sequencing metagenomic data.

[![Documentation Status](https://readthedocs.org/projects/metagensense/badge/?version=latest)](http://metagensense.readthedocs.io/en/latest/?badge=latest)


Citation
--------

  Correia D, Doppelt-Azeroual O, Denis JB et al., MetaGenSense: A web-application for analysis and exploration of high throughput sequencing metagenomic data [version 2; referees: 3 approved with reservations, https://f1000research.com/articles/4-86/v2]  F1000Research 2015.

Test MetaGenSense on a virtual machine
--------------------------------------

The **easiest way to test MetaGensense** is with the preconfigured virtual machine we provide. To use it:

1. Make sure [VirtualBox](https://www.virtualbox.org/wiki) is installed on your system
2. Download the MetaGenSense image at the following URL: http://webext.pasteur.fr/metagensense/metagensense.ova
3. Open it with Virtualbox. Once the virtual machine has started, the servers are automatically launched
4. Log in on the system, the login is `galaxy` and the password is `galaxy`
5. Launch Firefox (Web Browser icon, on the Desktop). It should automatically connect to the local MetaGenSense server (http://metagensense.test.fr:8000). To connect as a user, the login is `jdoe` and the password is `django`.
6. You can start using MetaGenSense by following the instructions included in the [documentation](http://metagensense.readthedocs.io/) to start a bioinformatics analysis. A test file is available on the desktop, `test.fastq`.

Install MetaGenSense v1.0 on your machine
-----------------------------------------

In case you would like to install MetaGensense directly on your machine, you can follow the installation instructions which are described step by step on the [installation section of the documentation](http://metagensense.readthedocs.io/en/latest/installation.html#installation).
